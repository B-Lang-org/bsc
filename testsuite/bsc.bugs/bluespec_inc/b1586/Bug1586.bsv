import Clocks::*;

module mkResetCombined();
   Clock c <- exposeCurrentClock;
   Reset r <- exposeCurrentReset;
   ReadOnly#(Bool) sInReset_val = isResetAsserted(clocked_by c, reset_by r);
endmodule

