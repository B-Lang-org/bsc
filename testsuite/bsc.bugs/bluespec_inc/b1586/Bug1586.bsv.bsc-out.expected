checking package dependencies
compiling Bug1586.bsv
Error: "Bug1586.bsv", line 6, column 35: (T0107)
  The type `ReadOnly' is not a module. Perhaps you have used the wrong
  function in a module instantiation, such as `map' vs `mapM'.
